src/app/gameModeName.js

```diff
@@ -0,0 +1,31 @@
+const createGameModeName = (modeName) => {
+    let gameModeNameDiv = document.createElement('div');
+    gameModeNameDiv.className = 'gameModeContainer';
+    
+    switch (modeName) {
+        case 'People':
+            gameModeNameDiv.id = 'people';
```

Comment:
Are you going to use those ids somewhere else later? They seem unnecessary now

Reply:
I had some bad experiences with stuff not having an identifier when it was needed, so I've decided to add it as it doesn't hurt to have it there :)

I've removed the id's as they probably won't be needed and are easy to be added